clojure-editors/editor-guides/setup.md

```diff
@@ -11,7 +11,7 @@ If anyone attending does not have access to a laptop running one of the above ch
 
```

Comment:
This is a redundant file that should be deleted instead.

Reply:
Yes I noticed that but I was not sure which one you would like to keep so I didn't touch it.

Reply:
Also please feel free to squash the commits in one commit when you decide to merge.